src/main/kotlin/io/snyk/plugin/services/SnykCliDownloaderService.kt

```diff
@@ -118,8 +141,11 @@ class SnykCliDownloaderService {
     }
 
     fun isNewVersionAvailable(currentCliVersion: String?, newCliVersion: String?): Boolean {
-        if (currentCliVersion == null || newCliVersion == null
-            || currentCliVersion.isEmpty() || currentCliVersion.isEmpty()) {
+        if (currentCliVersion == null ||
```

Comment:
Let's fix this detekt warning ("Complex conditions should be simplified and extracted into well-named methods if necessary")